sources/packages/web/src/components/layouts/aest/AESTHomeSideBar.vue

```diff
@@ -5,65 +5,40 @@
       density="compact"
       bg-color="background"
       active-color="primary"
-    >
-      <v-list-item
-        v-for="topItem in topItems"
-        :key="topItem.label"
-        :prepend-icon="topItem.icon"
-        :title="topItem.label"
-        :to="topItem.command()"
-      />
-    </v-list>
+      class="no-wrap"
+      :items="topItems"
+    />
     <v-list
       active-class="active-sidebar-item"
       density="compact"
       bg-color="background"
       active-color="primary"
-    >
-      <v-list-subheader class="nav-subtitle">Student requests</v-list-subheader>
-      <v-list-item
-        :prepend-icon="studentAccountApplicationItem.icon"
-        :title="studentAccountApplicationItem.label"
-        :to="studentAccountApplicationItem.command()"
-      />
-      <v-list-item
-        :prepend-icon="exceptionsItem.icon"
-        :title="exceptionsItem.label"
-        :to="exceptionsItem.command()"
-      />
-      <v-list-item
-        :prepend-icon="appealsItem.icon"
-        :title="appealsItem.label"
-        :to="appealsItem.command()"
-      />
-    </v-list>
+      class="no-wrap"
+      :items="studentRequestsItems"
```

Comment:
actually these 3 list an be merged to one (as its a single list), like this https://next.vuetifyjs.com/en/components/lists/#props (3rd example) with type property. insitution home side has the similar example
![image](https://user-images.githubusercontent.com/77353155/191111044-6d16b8c6-c1ed-48e9-9556-fe9cf0dcc1db.png)
![image](https://user-images.githubusercontent.com/77353155/191111082-0234c09a-0427-46af-97d0-0fcea85e5280.png)